docs/guides/miscellaneous/migration_guide.md

```diff
@@ -263,6 +315,80 @@ Suite Name                                   Status     Expectations met
 - Titanic.profiled                           ✔ Passed   2 of 2 (100.0 %)
 ```
 
+</TabItem>
+<TabItem value="spark">
+
+
+The example V2-style Checkpoint contains:
+  - A `LegacyCheckpoint`, with no `config_version` (versions were introduced as part of V3-style Checkpoints).
+  - A `validation_operator_name` that contains a reference to Validation Operators that are configured in the `great_expectations.yml` file, like `action_list_operator` in our example below.
+  - Reference to `batch_kwargs`, like in our example below.
+
+The example V3-style Checkpoint contains:
+  - A `Checkpoint` class with `config_version` populated (`1.0` in our example below).
+  - A list of `validations`, which contain [BatchRequests](/docs/reference/datasources#batches) that will be used to run the Checkpoint.
+  - A `action_list`, which contain a list of actions associated with the Validation Results (e.g., saving them for a later review, sending notifications in case of failures, etc.). These were known as Validation Operators in V2-style Checkpoints.
+
+:::note Migrating ExpectationSuites
+  
+  `ExpectationSuites` that were created in the V2-API will work in the V3-API **without** needing to be modified. However, `ExpectationSuites` also contain `metadata` describing the `batch` that was used to create the original `ExpectationSuite` object (under the `citations` field). For a suite that was created in V2, this metadata will contain `batch_kwargs`, and V3 suites will contain a `batch_request`. 
+  
+  If you choose to do so, the `citation` metadata can be migrated using the same pattern for migrating `batch_kwargs` to `batch_request` described below. 
+
+:::
+
+#### V2-Style Checkpoint
+
+```yaml file=../../../tests/test_fixtures/configuration_for_testing_v2_v3_migration/spark/v2/great_expectations/checkpoints/test_v2_checkpoint.yml#L1-L15
+```
+
+The Validation Operator named `action_list_operator` would be part of the `great_expectations.yml` file.
+
+```yaml file=../../../tests/test_fixtures/configuration_for_testing_v2_v3_migration/spark/v2/great_expectations/great_expectations.yml#L57-L69
+```
+
+#### V3-Style Checkpoint
+
+Here is the equivalent configuration in V3-style. Notice that the Validation Operators have been migrated into the `action_list` field in the Checkpoint configuration. In addition, you will also need to remove the Validation Operations from `great_expectations.yml` as a manual step.  Also, notice the `batch_request` that refers to the data asset rather than `batch_kwargs`.  In our example, there are also some additional parameters like `ge_cloud_id` and `expectation_suite_ge_cloud_id` that are added automatically and do not need to be modified as part of the migration.
```

Comment:
I think we should remove the last sentence regarding "additional parameters in our example [that can be ignored]"

Reply:
done